lib/api/api.js

```diff
@@ -450,20 +468,21 @@ api.prototype.getLatestInclusion = function(hashes, callback) {
 
         return self.getInclusionStates(hashes, Array(latestMilestone), callback);
     })
-}
+};
 
 /**
 *   Broadcasts and stores transaction trytes
 *
 *   @method storeAndBroadcast
 *   @param {array} trytes
-*   @returns {function} callback
+*   @param {apiResponseCallback} callback
 *   @returns {object} success
 **/
 api.prototype.storeAndBroadcast = function(trytes, callback) {
 
     var self = this;
 
+    // noinspection JSUnusedLocalSymbols
     self.storeTransactions(trytes, function(error, success) {
```

Comment:
Not sure if this syntax is understood universally by IDEs.  If not, we can remove it, but it helps to document that the unused `success` parameter is deliberately so.